apps/console/src/features/identity-providers/components/wizards/iproov/iproov-authentication-provider-create-wizard-content.tsx

```diff
@@ -0,0 +1,391 @@
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import { Field, Wizard, WizardPage } from "@wso2is/form";
+import { AxiosError } from "axios";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import {
+    IproovAuthenticationProviderCreateWizardFormValuesInterface
+} from "./iproov-authentication-provider-create-wizard";
+import { getIdentityProviderList } from "../../../api";
+import { IdentityProviderManagementConstants } from "../../../constants";
+import { IdentityProviderListResponseInterface, IdentityProviderTemplateInterface } from "../../../models";
+import { handleGetIDPListCallError } from "../../utils";
+
+/**
+ * Proptypes for the IproovAuthenticationWizardFrom.
+ */
+interface IproovAuthenticationProviderCreateWizardContentPropsInterface extends IdentifiableComponentInterface {
+    /**
+     * Trigger form submit.
+     * @param submitFunctionCallback - Callback.
+     */
+    triggerSubmission: (submitFunctionCallback: () => void) => void;
+    /**
+     * Trigger previous page.
+     * @param previousFunctionCb - Callback.
+     */
+    triggerPrevious: (previousFunctionCb: () => void) => void;
+    /**
+     * Callback to change the wizard page,
+     * @param pageNo - Page Number.
+     */
+    changePageNumber: (pageNo: number) => void;
+    /**
+     * IDP template.
+     */
+    template: IdentityProviderTemplateInterface;
+    /**
+     * Total wizard page count.
+     * @param pageCount - Page number.
+     */
+    setTotalPage: (pageCount: number) => void;
+    /**
+     * Callback to be triggered for form submit.
+     * @param values - Form values.
+     */
+    onSubmit: (values: IproovAuthenticationProviderCreateWizardFormValuesInterface) => void;
+}
+
+const FORM_ID: string = "iproov-authenticator-wizard-form";
+
+/**
+ * Iproov Authentication Provider Create Wizard content component.
+ *
+ * @param props - Props injected to the component.
+ * @returns Functional component.
+ */
+export const IproovAuthenticationProviderCreateWizardContent: FunctionComponent<
+    IproovAuthenticationProviderCreateWizardContentPropsInterface
+> = (
+    props: IproovAuthenticationProviderCreateWizardContentPropsInterface
+): ReactElement => {
+
+    const {
+        triggerSubmission,
+        triggerPrevious,
+        changePageNumber,
+        template,
+        setTotalPage,
+        onSubmit,
+        [ "data-componentid" ]: componentId
+    } = props;
+
+    const { t } = useTranslation();
+
+    const [ idpList, setIdPList ] = useState<IdentityProviderListResponseInterface>({});
+    const [ isIdPListRequestLoading, setIdPListRequestLoading ] = useState<boolean>(undefined);
+
+    /**
+     * Loads the identity provider authenticators on initial component load.
+     */
+    useEffect(() => {
+
+        getIDPlist();
+    }, []);
+
+    /**
+     * Get Idp List.
+     */
+    const getIDPlist = (): void => {
```

Comment:
Let's use `useIdentityProviderList` react hook [1] instead of this manual implementation if possible.

[1] https://github.com/wso2/identity-apps/blob/master/apps/console/src/features/identity-providers/api/identity-provider.ts#L148